### PR TITLE
CONSOLE-2371: [i18n] Externalize olm operand related strings

### PR DESCRIPTION
**Pull request by TheRealJon**
Externalize strings in `frontend/packages/operator-lifecycle-manager/src/components/operand`

### CODE REVIEW

File: frontend/packages/operator-lifecycle-manager/src/components/operand/DEPRECATED_operand-form.tsx
Comment (spadgett) [inline frontend/packages/operator-lifecycle-manager/src/components/operand/DEPRECATED_operand-form.tsx:840]: No need to localize this string

File: frontend/packages/operator-lifecycle-manager/src/components/operand/index.tsx
Comment (spadgett) [inline frontend/packages/operator-lifecycle-manager/src/components/operand/index.tsx:350]:
```suggestion
      title={t('olm~No provided APIs defined')}
```

File: frontend/packages/operator-lifecycle-manager/src/components/operand/index.tsx
Comment (spadgett) [inline frontend/packages/operator-lifecycle-manager/src/components/operand/index.tsx:408]: I would make `Create new` and `Create {{item}}` separate messages

File: frontend/packages/operator-lifecycle-manager/src/components/operand/utils.ts
Comment (spadgett) [inline frontend/packages/operator-lifecycle-manager/src/components/operand/utils.ts:174]: No need to localize

File: frontend/packages/operator-lifecycle-manager/src/components/operand/utils.ts
Comment (spadgett) [inline frontend/packages/operator-lifecycle-manager/src/components/operand/utils.ts:186]: No need to localize

File: frontend/packages/operator-lifecycle-manager/src/components/operand/utils.ts
Comment (spadgett) [inline frontend/packages/operator-lifecycle-manager/src/components/operand/utils.ts:209]: No need to localize

File: frontend/packages/operator-lifecycle-manager/src/components/operand/DEPRECATED_operand-form.tsx
Comment (rhamilto) [inline frontend/packages/operator-lifecycle-manager/src/components/operand/DEPRECATED_operand-form.tsx:1228]:
```suggestion
                  {t('public~Create')}
```

File: frontend/packages/operator-lifecycle-manager/src/components/operand/DEPRECATED_operand-form.tsx
Comment (rhamilto) [inline frontend/packages/operator-lifecycle-manager/src/components/operand/DEPRECATED_operand-form.tsx:1231]:
```suggestion
                  {t('public~Cancel')}
```

**Review by rhamilto — COMMENTED** [review]
`Create` and `Cancel` are already in `public` namespace, so no reason to add them to `olm`, too.

**Comment by TheRealJon** [comment]
/hold

For qe and docs approval

**Comment by rhamilto** [comment]
@TheRealJon rebase required.

**Review by spadgett — APPROVED** [review]
/lgtm

**Comment by spadgett** [comment]
/hold cancel

@TheRealJon FYI, needs rebase

**Comment by TheRealJon** [comment]
@spadgett rebased. Needs lgtm again.

**Review by spadgett — APPROVED** [review]
/lgtm

**Review by spadgett — APPROVED** [review]
/lgtm

**Pull request merged**
